open-metadata-implementation/access-services/cognos/cognos-server/src/main/java/org/odpi/openmetadata/accessservices/cognos/server/CognosServicesInstance.java

```diff
@@ -0,0 +1,41 @@
+/* SPDX-License-Identifier: Apache 2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.accessservices.cognos.server;
+
+
+import org.odpi.openmetadata.accessservices.cognos.assets.DatabaseContextHandler;
+
+/**
+ * CognosServicesInstance caches references to OMRS objects for a specific server.
+ * It is also responsible for registering itself in the instance map.
+ */
+public class CognosServicesInstance
```

Comment:
Needs to inherit from OMASSerivceInstance - or subclass

Reply:
implemented by https://github.com/odpi/egeria/pull/3044/commits/0c4d72f309938f86e2c3a8830dec3e2002e47a5f